src/utility/dynamicallyGeneratedEntries.js

```diff
@@ -0,0 +1,4 @@
+
+const dynamic = ['entities/Team.json', 'entities/Team_entries_en.json'];
```

Comment:
What's that?

Reply:
This is a list of files that should be left out when importing the agent because they are dynamically set via the API. In this case it's the list of teams, that is modified every time a team is created.

Reply:
Ah okay.
Maybe this should be some exclude or ignore file? 
Would be pretty cool if you could have an .agentexportignore file.